tests/end2end/features/utilcmds.feature

```diff
@@ -164,3 +164,7 @@ Feature: Miscellaneous utility commands exposed to the user.
     Scenario: Using debug-log-filter with invalid filter
         When I run :debug-log-filter blah
         Then the error "filters: Invalid value blah - expected one of: statusbar, *" should be shown
+
+    Scenario: Using debug-log-filter
+        When I run :debug-log-filter webview
+        Then no crash should happen
```

Comment:
We run all tests in one `.feature` file in the same qutebrowser instance for performance reasons, and most tests work by parsing qutebrowser's log - so this will probably break all tests following this one.

Even if we had a way to unset a filter with `:debug-log-filter`, the test would probably fail trying to execute that... so I don't really know a good solution for this. Do you?